documentation/experimental_features.md

```diff
@@ -8,6 +8,28 @@ API may change, requiring the user to update their code or configuration. The
 Bolt team attempts to make these changes painless by providing useful warnings
 around breaking behavior where possible. 
 
+## Streaming output
+
+This feature was introduced in [Bolt 3.2.0](https://github.com/puppetlabs/bolt/blob/main/CHANGELOG.md#bolt-310-2021-3-08).
+
+The new `stream` output option can be set in [transport
+configuration](bolt_transports_reference.md). Bolt will stream results back to the console as they
```

Comment:
It would be nice to have this is a CLI option as well, so it can be toggled easily. Should be as simple as adding a switch to the option parser since the logic for translating it to transport config is already there.

Reply:
Yeah - I also realized that now that it's added to all the transports (it was originally just Bash shell) it can also be a Bolt config option, instead of transport config. Refactored to suit.

Reply:
Ahh, that feels _much_ better than having it be a transport option. :) Good idea.